docs/commands/eject.md

```diff
@@ -0,0 +1,92 @@
+# eject
+
+## Introduction
+
+The `eject` command allows you to customize components by creating a local copy of their source code in your Redocly project.
+Use this feature when you need to modify a component's styles, structure, or behavior beyond what's possible through configuration.
+
+{% partial file="../_snippets/new-product-alert-commands.md" %}
+
+## Usage
+
+```bash
+redocly eject component <component-path>
+redocly eject component <component-path> [--force]
+redocly eject component <component-path> [--project-dir=<path>]
+redocly eject --help
+redocly eject --version
+```
+
+## Options
+
+| Option | Type | Description |
+|--------|------|-------------|
+| `component-path` | string | Path to the component or a glob pattern for multiple components. |
+| `--force`, `-f` | boolean | Skip the "overwrite existing" confirmation when ejecting a component that already exists in the destination. |
+| `--project-dir`, `-d` | string | Specifies the destination folder to eject components into. |
+| `--help` | boolean | Show help. |
+| `--version` | boolean | Show version number. |
+
+## Examples
+
+If no component is found when running the `eject` command, then it prints a list of ejectable components.
+
+### Eject a single component
+
+Eject an individual component by passing it's filepath.
+
+The following command ejects the `Filter` component:
```

Comment:
Where does it eject it to? What do I do next? Maybe add a sentence to prompt the user for their next move (could go after the code snippet)

Reply:
Added clarifying sentence afterwards